### ISSUE TITLE
can i use this code to train and test based on real-IAD directly now?

### ISSUE THREAD

**Issue by benshan123** [zhangzjn/ADer#31]
can i use this code to train and test based on real-IAD directly now?

**Comment by zhangzjn** [comment]
Certainly, we currently support the Real-IAD dataset. This can be achieved by modifying the `data.root` from 'data/mvtec' to 'data/realiad'.
